vm/Cargo.toml

```diff
@@ -78,9 +78,9 @@ result-like = "0.4.5"
 timsort = "0.1.2"
 
 # RustPython crates implementing functionality based on CPython
-sre-engine = "0.4.1"
+# sre-engine = "0.4.1"
 # to work on sre-engine locally or git version
-# sre-engine = { git = "https://github.com/RustPython/sre-engine", rev = "refs/pull/14/head" }
+sre-engine = { git = "https://github.com/RustPython/sre-engine", rev = "refs/pull/17/head" }
```

Comment:
I am sorry to forget about this. update to recent release of sre-engine or import the source code here?

Reply:
Can you move the repr to rustpython-sre-engine.

Reply:
I am going to import sre_engine in #5202